src/service/stockService.js

```diff
@@ -1,16 +1,23 @@
 'use strict';
 
-import StockRepository from '../repository/stockRepository.mjs';
 import _ from 'lodash';
+// import { container } from 'awilix';
```

Comment:
Why is this a comment?

Reply:
Had it import to test out dependency injection. Didn't need it anymore. I think it can be cleaned up by the next PR